docker-compose.yml

```diff
@@ -22,7 +23,7 @@ services:
       - matchbot
 
   db:
-    image: mysql:8.0
+    image: mariadb:11
```

Comment:
It's MySQL in prod, isn't it? Can you remind me what version? Would be good to keep dev as closely aligned to prod as we can. Would be good to know also about the reason for changing from MySQL to Maria

Reply:
Could we leave this as mysql 8 for now so we have dev closely aligned to prod? Or if we need to make them different can we add a ticker for bringing them back together soon?

Reply:
Done, with the caveat that I've added `platform` – would be good to confirm this doesn't break local runs for you on [x86?] Linux.

Reply:
That platform is the hardware platform I have, so doesn't require any emulation. Also tested and it works fine.